dashboard/app/models/contact_rollups_raw.rb

```diff
@@ -17,4 +17,27 @@
 
 class ContactRollupsRaw < ApplicationRecord
   self.table_name = 'contact_rollups_raw'
+
+  def self.truncate_raw_contacts
```

Comment:
This is a class method so it already knows the context it is running in. Also use `self.table_name` to avoid hard coding table name twice.

```suggestion
  def self.truncate_table
      ActiveRecord::Base.connection.execute("TRUNCATE TABLE #{self.table_name}")
  end
```